data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+			choice
+				`	"Yes, I can."`
+					accept
+				`	"I would rather not."`
+			`	Plume stoically accepts that with a nod "That is your choice. I will arrange for another ship to transport it."`
+			decline
+	on accept
+		event "remnant: penguin"
+	on complete
+		log "People" "Tali" `As an engineering prefect, Tali is the foremost authority among the Remnant on salvaged technology and a senior leader in their reverse engineering program. She is also a prominent engineer in the field of spacecraft propulsion.`
+		"remnant tali" ++
+		payment 1000000
+		conversation
+			branch tali
+				"remnant tali" > 1
+			`When you land on Viminal a solidly-built female Remnant in coveralls is waiting for you. "Greetings, Captain <first>. My name is Tali, and I am a Prefect among the Remnant. Above all else, I study motion, and in particular how our starships move."`
+			goto main
+			label tali
+			`	When you land on Viminal you spot Tali waiting for you. "Greetings, Captain <first>. It is good to see you again."`
+			label main
+			`	"So, Plume sent us some very interesting data, along with a message to expect you with a sample?" You hand her the container with the slime sample and she holds it up to the light. "Interesting looking stuff" she chants. "Let's go back to my shop." She leads the way through several side passages to an out-of-the-way landing pad connected to a large hanger. Inside, you can see a giant tank filled with a murky fluid, watched over by a flock of displays. She makes her way over to a large machine with a several slots which she slides the container into. As the machine starts to work, she turns back to you.`
+			`	"As the primary example of space-faring life, the Void Sprites have always been of interest to me. They have numerous advantages that we as atmosphere-bound creatures can only dream of, such as being able to experience space in a much more direct manner. I have been working on a side project to build a new generation of ships that melds these ideas with our current technology, but the data we have on the Void Sprites is centuries old and incomplete" her chant trails off as she pulls up a display with your original scans on them. "However, your recent explorations provided new data I could use to update my designs. This slime that you retrieved is the final ingredient."`
+			`	The machine dings, and she looks over the results and then trills excitedly. "Yes, it matches! So it is time to begin the final activation." She strides over to a central console and taps in a series of commands, followed by an iris scan and a blood prick. The industrial hum of the room suddenly ramps up to a crescendo as your senses are assaulted by what you can only describe as the smell of a thunderstorm in a swamp. A few seconds later it begins to quieten again, and the tank starts lowering into the floor as a shape emerges from the liquid.`
```

Comment:
`senses are assaulted` Technically only your noise is being assaulted by the smell. :)

Reply:
That's true. I was originally thinking about the noise also affecting one's ears.